handler/handler.go

```diff
@@ -223,6 +224,20 @@ func newHandler(fn interface{}) (Func, error) {
 			}
 			return out, nil, nil
 		}
+	default:
+		// A function that returns a value,function and an error.
```

Comment:
A function that returns a value, a callback function and an error.

Reply:
done.Thanks.